app/src/main/java/com/rkm/tasky/feature/common/Mode.kt

```diff
@@ -0,0 +1,7 @@
+package com.rkm.tasky.feature.common
+
+enum class Mode {
+    CREATE,
+    VIEW,
+    UPDATE,
+}
```

Comment:
Nit - It pays off in the long run to give the best names to parameters, classes, functions, and variables. I believe it is a genuinely underestimated part of writing code. Descriptive names help future developers onboard and understand better the intent of some code without making guesses or performing manual testing. That future developer could even be you coming back to a codebase that you haven't worked on for a couple of years 😄